packages/integrations/package.json

```diff
@@ -16,8 +16,10 @@
   "module": "esm/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
+    "@sentry/minimal": "5.20.1",
```

Comment:
I don't think we need the `@sentry/minimal` depedency.

Since we are in an integration, we have access to the hub with `getCurrentHub()`. This means we can use `hub.captureEvent()` - https://getsentry.github.io/sentry-javascript/classes/hub.hub-1.html#captureevent